File: 课程/CaiPiao/CaiPiao/STSettingItem.m
//
//  STSettingItem.m
//  CaiPiao
//
//  Created by kemchenj on 5/22/16.
//  Copyright © 2016 kemchenj. All rights reserved.
//

#import "STSettingItem.h"



@implementation STSettingItem

+ (instancetype)itemWithIcon:(UIImage *)icon title:(NSString *)title;
{
    STSettingItem *item = [[self alloc] init];

    item.icon  = icon;
    item.title = title;

    return item;
}


@end

File: 课程/彩票/彩票/Setting(设置)/Model/STSettingItem.swift
//
//  STSettingItem.swift
//  彩票
//
//  Created by kemchenj on 5/25/16.
//  Copyright © 2016 kemchenj. All rights reserved.
//

import Foundation
import UIKit

class STSettingItem: NSObject {
    
    var title : String?
    var subTitle : String?
    
    var icon  : UIImage?
    
    var operationBlock : ((indexPath: NSIndexPath)->())?
    
    init(title: String, icon: UIImage) {
        self.title = title
        self.icon = icon
        
        super.init()
    }
    
}